rules/prefer_to_be_null.js

```diff
@@ -8,31 +8,36 @@ const expectNotToBeCase = require('./util').expectNotToBeCase;
 const method = require('./util').method;
 const method2 = require('./util').method2;
 
-module.exports = context => {
-  return {
-    CallExpression(node) {
-      const is = expectToBeCase(node, null) || expectToEqualCase(node, null);
-      const isNot =
-        expectNotToEqualCase(node, null) || expectNotToBeCase(node, null);
+module.exports = {
+  meta: {
+    fixable: 'code',
```

Comment:
Wee, awesome! This is a change I wanted to make, nice that this yells at us

Reply:
should probably make all of the rules be an object with a `create` function, even if they don't have any other properties